hirola/front/templates/front/login.html

```diff
@@ -32,22 +43,27 @@ <h4>teke</h4>
                             <a href="#" class="right">Forgot password?</a>
                         </div>
                     </div>
-                    <div class="row">
+                    <div class="row center">
                         <button class="btn waves-effect waves-light bt light-green accent-4" type="submit" name="action">Login
                         </button>
                     </div>
-                    <div class="row">
-                        <div class="col s12">
-                            <h5 class="center">New customer?</h5>
+                    <div class="row center">
+                        <div class="row">
+                            <p class="center">Or register with your social account
+                            </p>
+                        </div>
+                        <div class="row social">
+                            <div class="col s6">
+                                <a href="#">
+                                    <i class="fa fa-google fa-lg right"></i>
+                                </a>
+                            </div>
+                            <div class="col s6">
+                                <a href="#">
+                                    <i class="fa fa-facebook-square fa-lg left"></i>
+                                </a>
+                            </div>
                         </div>
```

Comment:
I noted that in the design there was a signup link on the login page. I note it has been removed. I think it should be there to prevent a user who wanted to log in, is not a registered user and does not want to use the social media account to register to go back one page in order to do so.